README.md

```diff
@@ -29,6 +29,7 @@ These constructor arguments are available as attributes:
 - `N`: number of particles (int)
 - `box`: bounding box (`Box`)
 - `step`: timestep counter (int)
+- `num_types`: number of particle types (int)
```

Comment:
We should add a short comment (maybe at the end of this section) about what happens when `num_types is None` (i.e., the number of types is deduced from `typeid`).